### PR TITLE
Add image loaders to development env config.

### PR DESCRIPTION
**Pull request by julianorafael**
See #746

### CODE REVIEW

File: webpack.config.development.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ export default validate(merge(baseConfig, {
       { test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=application/octet-stream' },
       { test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, loader: 'file' },
       { test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=image/svg+xml' },
+      
+      {
+        test: /\.(?:ico|gif|png|jpg|jpeg|webp)$/,
+        loader: 'url-loader'
+      },
```
Comment (amilajack) [inline webpack.config.development.js]: Tests are failing because of this trailing comma

Comment (julianorafael) [inline webpack.config.development.js]: Just fixed it.

File: webpack.config.development.js [hunk 2]
```diff
@@ -49,6 +49,11 @@ export default validate(merge(baseConfig, {
       { test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=application/octet-stream' },
       { test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, loader: 'file' },
       { test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=image/svg+xml' },
+
```
Comment (jhen0409) [inline webpack.config.development.js]: https://travis-ci.org/chentsulin/electron-react-boilerplate/jobs/201818578#L396

Trailing spaces

**Comment by jhen0409** [comment]
Thanks! :)

**Pull request merged**